### ISSUE TITLE
Persistent code1 exits on macos 11.3/nova 6.2 system

### ISSUE THREAD

**Issue by jcogs-design** [alexanderweiss/nova-prettier#73]
I'm getting persistent errors running the Prettier extension in Nova 6.2 on a macos 11.3 system.
The extension log looks like this...
```
Prettier[11:28:59.200000] Loading bundled prettier at /Users/glawrie/Library/Application Support/Nova/Extensions/alexanderweiss.prettier/node_modules/prettier

Prettier[11:28:59.201000] Starting Prettier service

Prettier[11:28:59.558000] Prettier service exited with code 1

Prettier[11:28:59.559000] Starting Prettier service

Prettier[11:28:59.921000] Prettier service exited with code 1
```
I've uninstalled and reinstalled the extension but it makes no difference.
Any ideas what I can do to get it to work?

**Comment by alexanderweiss** [comment]
Hey! What version of node do you have? I'm not 100% sure what version Prettier requires, but it can't be too old. You might want to double-check the one that Prettier uses:

1. Go to the extension library
2. Right click the Prettier extension and then Show in Finder
3. Open that folder (called alexanderweiss.prettier) in Terminal
4. Run /usr/bin/env node -v

**Comment by jcogs-design** [comment]
Haha - that was it.  Was on version 11.15.  Have updated (now on 14.16.1) and extension is working again.  Many thanks!

**Comment by jeremyorion** [comment]
FWIW, I ran into this today running node v15.14.0 (Mac 11.13.1, Nova 6.2).

No luck trying different node versions, but then noticed my error was specific a single Nova project (that is, the Prettier extension worked fine in other projects). I recreated said project, cautiously stepping through installing various node packages one-by-one. The extension continued to work even after a complete recreation. 

Thought this worth adding here, in case others run into something similar

**Comment by alexanderweiss** [comment]
Yeah, I've seen that before. No clue what the exact cause of that is. I'm currently working on logging the errors when trying to import the prettier module though. So that should help next time.
